src/components/NewsCard/index.jsx

```diff
@@ -3,24 +3,17 @@ import styles from './styles.module.css';
 import Image from '@theme/IdealImage';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faCalendar } from '@fortawesome/free-regular-svg-icons'
-import CondaCSVG from '@site/static/img/conda_c.svg';
+import Link from '@docusaurus/Link';
 
-
-export default function NewsCard(props) {
-  const url = `/blog/${props.slug}`;
-  let image = "";
-  
-  if (props.imageUrl) {
-    image = (
-    <div>
-      <Image img={props.imageUrl} alt={`Banner image for ${props.title}`} className={styles.news_card_image} /> 
-      <hr style={{backgroundColor: "var(--ifm-color-secondary-lightest"}} />
-    </div>)
-  } else if (props.index === 0) {
-    image = (
-      <CondaCSVG />
-    )
-  }

```

Comment:
This is why we get the big C if no image is defined in the frontMatter.

Reply:
Opted to remove this, it looks cleaner IMO to display the image if present and to have no image otherwise